### PR TITLE
Added missing builld components to RCD [empty]

### LINKED ISSUE #28278: RCD in CE's room on Box station
## Description
The RCD on the table of CEs room lacks comps and updated strings, its unable to deconstruct things as well as make wires

(This may be an issue with other RCD's as well)
**Reproduction**
1 Spawn on Box as CE.
2 Take RCD thats on the table
3 suffer

**Screenshots**
image
on the left is the one that spawns in on the table, the right is one i spawned from the entity menu.

### PR DESCRIPTION
**Pull request by JIPDawg**
## About the PR

Added missing builds to the RCD[Empty]. This matches with the rest of the RCDs

## Why / Balance

fixes #28278
## Media

- [ x ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Review by Brandon-Huu — CHANGES_REQUESTED** [review]
https://github.com/space-wizards/space-station-14/blob/3b914e33ffc3c05ae5fb4fbd159668d3dd5cffd8/Resources/Prototypes/Entities/Objects/Tools/tools.yml#L404-L424

Why can't you just remove the RCD component from RCDEmpty?

**Comment by Plykiya** [comment]
Yeah, it doesn't need to have the AvailablePrototypes declared twice. It already parents the original RCD.

**Review by Tayrtahn — CHANGES_REQUESTED** [review]
As the others said, simply remove the RCD component from the empty RCD prototype. That way it will inherit the component and its values from the parent.

**Comment by JIPDawg** [comment]
That's why they pay you guys the big bucks :godo:

File: Resources/Prototypes/Entities/Objects/Tools/tools.yml [hunk 1]
```diff
@@ -401,20 +401,6 @@
   components:
   - type: LimitedCharges
     charges: 0
```
Comment (Plykiya) [inline Resources/Prototypes/Entities/Objects/Tools/tools.yml:403]: Max charges needs to be restated to 30 since the component is being redeclared

Comment (Tayrtahn) [reply Resources/Prototypes/Entities/Objects/Tools/tools.yml:403]: It doesn't, actually. Top-level component fields are inherited.

**Review by Tayrtahn — APPROVED** [review]

**Pull request merged**